src/protocol.vala

```diff
@@ -318,7 +318,12 @@ namespace LanguageServer {
 
     class CompletionList : Object, Json.Serializable {
         public bool isIncomplete { get; set; }
-        public Gee.List<CompletionItem> items { get; private set; default = new Gee.LinkedList<CompletionItem> (); }
+        public Gee.Collection<CompletionItem> items { get; private set; }
+
+        public CompletionList.with_items (Gee.Collection<CompletionItem> items) {
```

Comment:
To keep the purpose of this merge commit clear, please don't change anything else that is not strictly pertinent to the issue.